index.html

```diff
@@ -28,6 +28,131 @@ <h1><span class="greetings">Hey there. I’m Josue</span><br>
+            <li class="language">Ruby on rails</li>
+            <li class="language">css</li>
+            <li class="language">html</li>
+            <li class="language">JavScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button></span>
+      </article>
+      <article class="project-template">
+        <span class="place-holder">
+          <h3>Multi-Post Stories <br>Gain+Glory</h3>
+          <ul class="pr-language">
+            <li class="language">Ruby on rails</li>
+            <li class="language">css</li>
+            <li class="language">html</li>
+            <li class="language">JavScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button></span>
+      </article>
+      <article class="project-template">
+        <span class="place-holder">
+          <h3>Multi-Post Stories <br>Gain+Glory</h3>
+          <ul class="pr-language">
+            <li class="language">Ruby on rails</li>
+            <li class="language">css</li>
+            <li class="language">html</li>
+            <li class="language">JavScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button></span>
+      </article>
+      <article class="project-template">
+        <span class="place-holder">
+          <h3>Multi-Post Stories <br>Gain+Glory</h3>
+          <ul class="pr-language">
+            <li class="language">Ruby on rails</li>
+            <li class="language">css</li>
+            <li class="language">html</li>
+            <li class="language">JavScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button></span>
+      </article>
+      <article class="project-template">
+        <span class="place-holder">
+          <h3>Multi-Post Stories <br>Gain+Glory</h3>
+          <ul class="pr-language">
+            <li class="language">Ruby on rails</li>
+            <li class="language">css</li>
+            <li class="language">html</li>
+            <li class="language">JavScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button></span>
+      </article>
+      <article class="project-template">
+        <span class="place-holder">
+          <h3>Multi-Post Stories <br>Gain+Glory</h3>
+          <ul class="pr-language">
+            <li class="language">Ruby on rails</li>
+            <li class="language">css</li>
+            <li class="language">html</li>
+            <li class="language">JavScript</li>
+          </ul>
+          <button class="button" type="button">See Project</button></span>
+      </article>
+             <!-- About me project part -->
+    <h2 class="titles-about-me">About me</h2>
+    <p class="paragraph">
+      I can help you build a product , feature or website Look through
+      some of my work and experience! If you like what you see and have a
+      project you need coded, don’t hestiate to contact me.
```

Comment:
- [ ] Please, try to stick as much as possible to the Figma design 🎨 , at the moment you're missing a background image for your about section and also some styling on the paragraph, kindly fix this. 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/198692096-816fff11-339f-461b-aaf7-9de9db99957b.png) | ![image](https://user-images.githubusercontent.com/77069011/198692047-7303d987-f8e6-43b3-b612-24e05b7b8d20.png) |